ttfb.sql

```diff
@@ -11,6 +11,8 @@ DECLARE PLATFORMS ARRAY<STRUCT<regex STRING, name STRING>> DEFAULT [
   ('wordpress.com', 'Automattic'),
   ('a9130478a60e5f9135f765b23f26593b', 'Automattic'),
   ('flywheel', 'Flywheel'),
+  ('caddy/', 'Framer'),
```

Comment:
Is "Caddy" to do with CaddyServer? Are those ALWAYS framer websites?

Reply:
Caddy indeed has something to do with Caddy :)

By default Caddy uses just `Server: Caddy`, we're returning `Caddy/opaque-versiony-thing` at the moment. In hindsight that might sound a bit like a bad choice, we used it though to recognize the platform while we're switching. 

Plan is to go back to just `Framer/` as prefix, as soon as we've done that I can file an update to drop this one again.

Reply:
OK I ran the below query to compare this header with Wappalyzer detection (that doesn't use this header) and seems like the vast majority of these are Framer sites (and the others might well be false negatives). So LGTM.

<img width="451" alt="image" src="https://github.com/rviscomi/ismyhostfastyet/assets/10931297/52818660-bd13-4c6b-aad0-ad1340286a69">
 

```SQL
SELECT
  client,
  COUNTIF(p.page IS NOT NULL) as framer_pages,
  COUNTIF(p.page IS NULL) as not_framer_pages
FROM
  `httparchive.all.requests` r
LEFT OUTER JOIN
  (
  SELECT date, client, page, root_page
  FROM
    `httparchive.all.pages` p,
  UNNEST (technologies) as tech
  WHERE date = "2024-03-01" AND
  tech.technology = 'Framer Sites'
  ) p
USING (date, client, page, root_page)
WHERE
  date = "2024-03-01" AND
  is_main_document AND
  LOWER(CONCAT(IFNULL(JSON_VALUE(r.summary, '$.resp_x_powered_by'), ''), IFNULL(JSON_VALUE(r.summary, '$.resp_via'), ''), IFNULL(JSON_VALUE(r.summary, '$.resp_server'), ''))) LIKE 'caddy/%'
GROUP BY client
```